aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -366,18 +384,24 @@ static void slow_conv2d_backward_weight_out_cpu_template(
 
   auto input = input_.contiguous();
   auto grad_output = grad_output_.contiguous();
+  TORCH_CHECK(grad_weight.is_contiguous(), "slow_conv2d: grad_weight must be contiguous");
```

Comment:
this is checked earlier, in line 369

Reply:
You also need to check that finput is contiguous?

Reply:
Changed to check `finput`.